src/main/java/com/develhope/spring/vehicles/optionals/models/StartAndStop.java

```diff
@@ -0,0 +1,51 @@
+package com.develhope.spring.vehicles.optionals.models;
+
+import jakarta.persistence.*;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+
+@Entity
+@Table(name = "Start&Stop")
+public class StartAndStop {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private boolean installed;
```

Comment:
non servono booleani per ora, piuttosto serve una variabile Vehicle vehicle con relazione one to one se vuoi già aggiungerla per il futuro